fvm/blueprints/scripts/systemChunkTransactionTemplate.cdc

```diff
@@ -19,9 +20,12 @@ transaction {
             ?? panic("Couldn't borrow RandomBeaconHistory.Heartbeat Resource")
         randomBeaconHistoryHeartbeat.heartbeat(randomSourceHistory: randomSourceHistory())
 
-        let evmHeartbeat = serviceAccount.storage.borrow<&EVM.Heartbeat>(from: /storage/EVMHeartbeat)
-        if evmHeartbeat != nil { // skip if not available 
-            evmHeartbeat!.heartbeat()
-        }
+        let evmHeartbeat = serviceAccount.storage
+            .borrow<&EVM.Heartbeat>(from: /storage/EVMHeartbeat)
+        evmHeartbeat?.heartbeat()
```

Comment:
I spent way to long looking for a `'.` operator in the cadence language docs only to find out there is something wrong with the fonts:
![image](https://github.com/user-attachments/assets/576e9c44-7b50-4298-9b0b-e2f29cd75881)

Reply:
Where is this screenshot from? GitHub? VS Code? It's probably not a problem with the PR though?

Reply:
This is github on firefox. No not an issue in the PR. Just some sort of font issue somewhere.